pennylane/interfaces/keras.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the :func:`to_keras` function to convert quantum nodes into Keras layers.
+"""
+import tensorflow as tf
+import pennylane as qml
+
+major, minor, patch = tf.__version__.split(".")
+
+if (int(major) == 1 and int(minor) < 4) or int(major) == 0:
+    raise ImportError("Must use TensorFlow v1.4.0 or above")
```

Comment:
PennyLane only supports TF 1.12 and above due to eager execution mode :)

However, I think it should be fine if you want to make this TF 2.0 compatible only (since the API changed significantly between 1.X and 2.0)